src/pages/game/lib/Scene.tsx

```diff
@@ -0,0 +1,127 @@
+import type { NextPage } from 'next';
```

Comment:
Move this file to `/dto` instead as not exporting React Component (e.g. `<p>hello</p>`) as default will raise errors.